### PR TITLE
updated to working url for clip.mp4

### PR DESCRIPTION
**Pull request by jaburges**

### CODE REVIEW

File: docs/docs/usage/api.md [hunk 1]
```diff
@@ -172,7 +172,7 @@ Accepts the following query string parameters, but they are only applied when an
 | `timestamp` | int  | Print the timestamp in the upper left (0 or 1)    |
 | `crop`      | int  | Crop the snapshot to the (0 or 1)                 |
 
-### `/clips/<camera>-<id>.mp4`
+### `/api/frigate/notifications/<id>/<camera>/clip.mp4`
```
Comment (paularmstrong) [inline docs/docs/usage/api.md:175]: This API is the HTTP API that is unrelated to the Home Assistant API and the docs are correct per this spec.

It looks like we don't have a detailed explanation in the [Home Assistant API](https://blakeblackshear.github.io/frigate/usage/home-assistant#api) page of how to get to clips/etc. If you'd like to expand that, please do!

**Review by paularmstrong — CHANGES_REQUESTED** [review]

File: docs/docs/usage/api.md (hunk 1 above)
Comment (jaburges) [reply docs/docs/usage/api.md:175]: ohh i see - sorry, you are right - i'm using it from Node-Red to send a clip
I actually couldnt get /clips/camera-id.mp4 to work (but its not exposed outside my network so wouldnt matter anyway) 
i'll close and take a look at the Homeassistant docs
